docs/tokenizers.rst

```diff
@@ -0,0 +1,32 @@
+Tokenizers
+===========
+
```

Comment:
It might be useful here to add an introductory paragraph explaining why tokenizers could be useful for DeepChem models (perhaps linking to the tutorial for ChemBerta) and providing a bit of broader context for why tokenizers could be powerful for other scientific applications like genomics etc.

Reply:
done

Reply:
Perhaps we could clarify a bit here. Something like 

"A tokenizer is in charge of preparing the inputs for a natural language processing model. For many scientific applications, it is possible to treat inputs as "words"/"sentences" and use NLP methods to make meaningful predictions. For example, SMILES strings or DNA sequences have grammatical structure and can be usefully modeled with NLP techniques. DeepChem provides some scientifically relevant tokenizers for use in different applications. These tokenizers are based on those from the Huggingface transformers library (which DeepChem tokenizers inherit from)."